reverb/pip_package/build_pip_package.sh

```diff
@@ -71,6 +71,19 @@ function prepare_src() {
   # must remain where they are for TF to find them.
   find "${TMPDIR}/reverb/cc" -type d -name ops -prune -o -name '*.so' \
     -exec mv {} "${TMPDIR}/reverb" \;
+
+  # Copy darwin libs over so they can be loaded at runtime
+  so_lib_dir=$(ls $RUNFILES | grep solib) || true
```

Comment:
this code strikes me as a little fragile, using solib etc.  can add the solib directly to the ls?

same in below's ls statement.

Reply:
ditto